src/app/events/event-edit/event-edit.html

```
@@ -2,72 +2,92 @@
-    [matMenuTriggerFor]="menu"
-  >
-    <fa-icon [icon]="faEllipsisVertical" />
-  </button>
-  <mat-menu #menu="matMenu">
-    <!--    <button mat-menu-item (click)="updateVisibility()">-->
-    <!--      Update visibility-->
-    <!--    </button>-->
-  </mat-menu>
+  @if (eventQuery.isSuccess()) {
+    <button
+      title="menu"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)